File: URI-py/1117.py
# -*- coding: utf-8 -*-
qtde = 0
nota = 0
while qtde < 2:
    x = float(input())
    if x >= 0 and x <= 10:
        nota += x
        qtde += 1
    else:
        print("nota invalida")
print("media = %.2f"%(nota/2))

File: URI-cpp/1117.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    int n=0;
    double sum,result,nota;
    while(true){
        if(n==2){
            break;
        }
        cin >> nota;
        if(nota<0 || nota>10){
            cout << "nota invalida" << endl;
        } else {
            sum += nota;
            n++;
        }
    }

    result = sum/2.0;

    cout << "media = " << fixed << setprecision(2) << result << endl;

    return 0;
}
